### PR TITLE
vault 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by sidick**
Add formula for vault 0.1.0

### CODE REVIEW

File: Library/Formula/vault.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  version "0.1.0"
+  sha256 "e0a1a1ace4c9577408800825b374c0239c04ebc9ab6ed18c924dc8f0393b9fd5"
+
+  def install
+    bin.mkpath
+    bin.install "vault"
+  end
+
+  test do
+    system bin/"vault", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/vault.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/vault.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  url "https://dl.bintray.com/mitchellh/vault/vault_0.1.0_darwin_amd64.zip"
```
Comment (mikemcquaid) [inline Library/Formula/vault.rb]: This needs to be built from source.

File: Library/Formula/vault.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  version "0.1.0"
+  sha256 "e0a1a1ace4c9577408800825b374c0239c04ebc9ab6ed18c924dc8f0393b9fd5"
+
+  def install
+    bin.mkpath
```
Comment (mikemcquaid) [inline Library/Formula/vault.rb]: `bin` is created by `bin.install` so this line is unnecessary.

File: Library/Formula/vault.rb (hunk 1 above)
Comment (sidick) [inline Library/Formula/vault.rb]: OK, will find something, thanks for the tips!

File: Library/Formula/vault.rb (hunk 2 above)
Comment (sidick) [inline Library/Formula/vault.rb]: Will do

**Comment by sidick** [comment]
Closing this as it seems the source repo currently doesn't tag the releases.

**Comment by dunn** [comment]
They just tagged a 1.0 release: <https://github.com/hashicorp/vault/releases/tag/v0.1.0>

**Comment by sidick** [comment]
Yep, I raised it add ask issue with them. Didn't expect that quick a response :)

**Comment by egrim** [comment]
I'm considering creating a built-from-source formula for vault but don't want to duplicate effort.  @sidick: are you (or is anyone else) currently working on it?

**Comment by sidick** [comment]
I'm planning to try it this weekend but if you want to it'll probably be quicker as I've not done one before

**Pull request closed without merging**